crates/holochain_types/src/app.rs

```diff
@@ -546,20 +558,42 @@ impl InstalledAppCommon {
             .ok_or_else(|| AppError::RoleNameMissing(role_name.clone()))
     }
 
+    /// Accessor for particular role
```

Comment:
Please can this doc comment reference `AppRoleAssignment::Primary` so that the docs for what `primary` means are easily discoverable here?